service_discovery/samples/simple-discovery/provider/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "provider"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
Please add the license header, and add `license = "MIT"` under `edition = "2021"`

You can remove this comment.